ql/src/test/queries/clientpositive/stat_estimate_drill.q

```diff
@@ -25,4 +28,8 @@ explain analyze select sum(a) from t3 where a=1 group by b;
 explain analyze select sum(a) from t3 where a=1 and b=2 group by b;
 explain analyze select sum(a) from t3 where a=1 and b=2 and c=3 group by b;
 
+
+-- FIXME: for the below case; we right now get an IN which is estimated as 1/3 - might worth consider improving
```

Comment:
This is kind of surprising... IN clauses are handled in StatsRulesProcFactory, including IN + STRUCT.